config/op-consumer/mapping/queries/op2dl/pass/pass-without-identifier-or-classsification.sparql

```diff
@@ -1,27 +1,28 @@
 CONSTRUCT {
-  ?s ?p ?o;
-    a <http://www.w3.org/ns/org#Organization> . 
-}  
+  ?s ?p ?o .
```

Comment:
To understand what's coming from OP, there are `org:Organizations` but the bestuurseenheden are not `org:Organizations`?

Reply:
So I double checked. It seems like in the landing zone, the only Agent and Organization types we have are linked to ROs. But it's a bit risky to assume OP will not be producing the Agent extra type someday because Organization inherits from Agent...